elastic/kibana
# [UII] Add warning about unprivileged agents next to agent count

## Summary

Resolves https://github.com/elastic/ingest-dev/issues/3352.

If there are unprivileged agents enrolled in an agent policy that requires root privileges, this PR adds a warning icon and a tooltip next to the agent count for that policy:

<img width="1198" alt="image" src="https://github.com/elastic/kibana/assets/1965714/76fc9c7b-4c46-4f1d-ba2f-d4c9af14884b">

<img width="1403" alt="image" src="https://github.com/elastic/kibana/assets/1965714/5d1f0831-7e66-47ff-8bec-4dbdb13b90ab">

### Testing
1. enroll an agent with docker (it has unprivileged: true)
2. try to add an integration that requires root e.g. `auditd_manager` to its agent policy
3. verify that the warning icon and tooltip is visible for that policy in the list and detail views
